### DISCUSSION TITLE
香菜满园 | /cn/2019/05/cilantro-garden/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1337] [Comments]
https://yihui.org/cn/2019/05/cilantro-garden/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
大蒜大葱都种了，洋葱还没种过，下次可以试试。种了三年大蒜，直到前一阵子才知道蒜薹要掐，不然蒜瓣长不壮实，而且我之前也不知道蒜薹什么时候适合吃，等我问的时候，它们都已经硬化成结实的棍子。

今年这边天气不行，菜都长得慢。辣椒我也种了，但到现在才二三十厘米高，花才开了一两朵，都快被周围的西瓜蔓给盖住了；西红柿现在也才刚开始挂小果。只有黄瓜天天结天天吃。

> ——原帖发布于 2019-07-26 14:14:14

**Reply by yihui** [reply]
我这只是报了一部分菜名，剩下的还有韭菜、苦瓜、黄瓜、红薯、莴苣、南瓜、香瓜、芹菜、冬瓜、秋葵、紫苏……

> ——原帖发布于 2019-07-28 06:32:27

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
主要是前一年冬天没有把盆搬进室内，受不了本州冬天的苦寒，基本上被团灭了，第二年春天努力活过来一棵，但盆里又被我种了红薯，最后那仅存的一棵又打不过红薯，就这样彻底灭绝了。今年再找人寄了几根过来，但不知为什么唯独这封信没有信件通知，导致我几天没去看邮箱，等我去看的时候，已经在信箱里躺了好几天了，奄奄一息。伺候了快两个月，昨天终于看见貌似有一片叶子长出来了。

> ——原帖发布于 2021-06-06 03:20:25

[a comment by an automated account is not shown]

**Comment by licw58** [comment]
昨天给二娃读了一个童谣故事，《一园青菜成了精》，就想起了你的菜园。你能把鱼腥草养挂了，而我特意种的香菜从来就没发芽过，不知是每次都碰巧买了假种子还是我家的土壤不适合香菜种子发芽。

**Reply by yihui** [reply]
就像别人无法想象鱼腥草怎么会死一样，我也无法想象香菜怎么会不发芽。我每年都是一把种子乱撒在园子里，任它们自己发芽。气温降到零度后，就只剩香菜还能活着。更冷的时候它们也会冻死，但来年春天还一样会重新长出来。

**Reply by licw58** [reply]
我种香菜是几年前的事情了。依约记得当时我妈说香菜是夏天播种，秋天生长，冬天收获，所以极有可能是晒死或热死了。另外我种的香菜跟你的好像不是一个品种，是紫根矮茎匍匐生长型，成品超不过20厘米。我妈说这种香味馥郁，不足之处是生长极慢。
种菜种花看天意，我日夜频顾惜的牡丹去年也死了，不知是晒死还是旱死，打算扔掉的芍药倒是开了极美丽的花儿。

**Reply by yihui** [reply]
我这个香菜品种不太行，它太容易长茎了，我希望香菜可以多长点叶子。

去年的天意比较眷顾我的茄子和木耳菜，尤其是木耳菜，也是成了精，后来我都没精力去收服它，要是像老家养头猪就方便了。

**Reply by licw58** [reply]
这篇文章勾起了我对香菜的回忆。现在菜市场上成捆卖的香菜（跟你种的很类似）对比我小时候吃的紫根矮茎香菜，就跟你有篇文章里提到的现在的西红柿、黄瓜跟小时候的味道的差异。现在的香菜品种适合包饺子，香菜猪肉馅。

你又提到养猪，哈哈，挺期待你能说服邻居、物业，达成所愿。我曾经短暂养过一只鸡，一只兔。但是养动物太操心了，比如，夏天突然狂风暴雨电闪雷鸣，我在外面还得冒雨赶回家把兔子放屋里，怕它被雷神吓着。后来不堪所累，就送人了。

**Reply by yihui** [reply]
小区里面养猪养鸡是不可能的。唯一的可能是将来退休了买个农场。不过我对养家畜的兴趣并不大，我已经不是肉食动物了，也不想操这份心。

**Reply by licw58** [reply]
是我理解错了，我先入为主地以为你要像养宠物一样养家畜。不养的好，操心的事越少越好。
